BotModules/SteamBot/index.js

```diff
@@ -28,21 +28,17 @@ module.exports = function(bot) {
   });
   
   function PickRandomSharedGame(msg, mentions){
-    GetGamesList(mentions).then(games =>{
+    MongoDB.GetCommonGames(mentions).then(games =>{
       let rdmIdx = Math.floor(Math.random() * games.length)
       msg.channel.send(`Lets play ${games[rdmIdx].name}!`)
     })
```

Comment:
@xtr3m3nerd had wanted to add db close here, but I'm not sure I can because of what GetCommonGames is returning. Anything come to mind?

Reply:
Add a function to the MongoDB libery called Close and call MongoDB.Close() in the .then

Reply:
See c70f4ee and lmk if anything needs changing